lib/system/web/src/dom/dom_container.rs

```diff
@@ -1,16 +1,17 @@
-// This file contains the implementation of DOMContainer. A struct that aids us to handle html
-// elements, get its position and dimension avoiding style reflow.
-//
-// It relies on Resize Observer and Intersection Observer, which notifies us when the element's
-// rect and visibility rect is updated.
+//! This file contains the implementation of DOMContainer. A struct that aids us to handle html
+//! elements, get its position and dimension avoiding style reflow.
+//!
+//! It relies on Resize Observer and Intersection Observer, which notifies us when the element's
+//! rect and visibility rect is updated.
 
-use crate::prelude::*;
+use basegl_prelude::*;
 
-use crate::system::web::get_element_by_id;
-use crate::system::web::dyn_into;
-use crate::system::web::Result;
-use crate::system::web::StyleSetter;
-use crate::system::web::intersection_observer::IntersectionObserver;
+use crate::get_element_by_id;
+use crate::dyn_into;
+use crate::Result;
```

Comment:
What it is? Where it is defined?

Reply:
@wdanilo please note that I moved `dom_container` to `system/web` crate.